lib/pf/file_paths.pm

```diff
@@ -112,6 +114,7 @@ $log_conf_dir  = catdir( $conf_dir,"log.conf.d" );
 $generated_conf_dir   = catdir( $var_dir,"conf");
 $tt_compile_cache_dir = catdir( $var_dir,"tt_compile_cache");
 $pfconfig_cache_dir = catdir( $var_dir,"cache/pfconfig");
+$domains_chroot_dir = catdir( "/chroots");
```

Comment:
Wouldn't
catdir( $var_dir,"chroots");
Be better?
Or are you afraid @louismunro may delete everything under var